vcd/terraform_binary_test.go

```diff
@@ -1,4 +1,4 @@
-//go:build binary
+//go:build binary && !tm
```

Comment:
The test templates files that are in `vcd/test-templates` are not compatible with TM therefore this is not being executed when `tm` tag is specified